src/epm.rs

```diff
@@ -31,34 +31,46 @@ use subxt::{dynamic::Value, rpc::rpc_params, tx::DynamicTxPayload};
 const EPM_PALLET_NAME: &str = "ElectionProviderMultiPhase";
 
 #[derive(Copy, Clone, Debug)]
-struct EpmConstant {
-	epm: &'static str,
+struct Constant {
+	pallet: &'static str,
 	constant: &'static str,
 }
 
-impl EpmConstant {
-	const fn new(constant: &'static str) -> Self {
-		Self { epm: EPM_PALLET_NAME, constant }
+impl Constant {
+	const fn new(pallet: &'static str, constant: &'static str) -> Self {
+		Self { pallet, constant }
 	}
 
 	const fn to_parts(&self) -> (&'static str, &'static str) {
-		(self.epm, self.constant)
+		(self.pallet, self.constant)
 	}
 
 	fn to_string(&self) -> String {
-		format!("{}::{}", self.epm, self.constant)
+		format!("{}::{}", self.pallet, self.constant)
 	}
 }
 
 /// Read the constants from the metadata and updates the static types.
 pub(crate) async fn update_metadata_constants(api: &SubxtClient) -> Result<(), Error> {
-	const SIGNED_MAX_WEIGHT: EpmConstant = EpmConstant::new("SignedMaxWeight");
-	const MAX_LENGTH: EpmConstant = EpmConstant::new("MinerMaxLength");
-	const MAX_VOTES_PER_VOTER: EpmConstant = EpmConstant::new("MinerMaxVotesPerVoter");
+	const SIGNED_MAX_WEIGHT: Constant = Constant::new(EPM_PALLET_NAME, "SignedMaxWeight");
+	const MAX_LENGTH: Constant = Constant::new(EPM_PALLET_NAME, "MinerMaxLength");
+	const MAX_VOTES_PER_VOTER: Constant = Constant::new(EPM_PALLET_NAME, "MinerMaxVotesPerVoter");
+	const EPOCH_DURATION: Constant = Constant::new("Babe", "EpochDuration");
+	const SESSIONS_PER_ERA: Constant = Constant::new("Staking", "SessionsPerEra");
+
+	// TODO: no constant is exposed directly for this, hardcoded for now.
+	const SLOT_DURATION_SECS: u32 = 6;
```

Comment:
//cc @kianenigma I only found [this](https://github.com/paritytech/polkadot/blob/master/runtime/westend/constants/src/lib.rs#L44) which is exposed indirectly via [MinimumPeriod](https://github.com/paritytech/polkadot/blob/master/runtime/westend/src/lib.rs#L288) which isn't what I want...

Thus, hardcoded it